web/client/codechecker_client/cmd_line_client.py

```diff
@@ -954,13 +991,19 @@ def print_reports(client, reports, output_format):
                         source_line_contents[report.fileId][bug_line])
             rows.append(
                 (sev, checked_file, check_msg, check_name, source_line))
+
+            severity_stats[sev] += 1
+            file_stats[file_name] += 1
+
         if output_format == 'plaintext':
             for row in rows:
                 print("[{0}] {1}: {2} [{3}]\n{4}\n".format(
                     row[0], row[1], row[2], row[3], row[4]))
         else:
             print(twodim_to_str(output_format, header, rows))
 
+        print_stats(reports, file_stats, severity_stats)
```

Comment:
Do we print these stats after the json output too?

Reply:
No, we do not print the statistics after the json output. The json output handled in a different branch:
https://github.com/Ericsson/codechecker/blob/427c1ff3aa12c01c1e9ac4115e1cc91ae37d344f/web/client/codechecker_client/cmd_line_client.py#L884-L893